spec/src/main/asciidoc/required-apis.asciidoc

```diff
@@ -23,19 +23,19 @@
 
 MicroProfile implementations must include the following specifications:
 
- - <<jakarta-cdi, CDI (Jakarta Contexts and Dependency Injection) 2.0>>
- - <<jakarta-jaxrs, JAX-RS (Jakarta RESTful Web Services) 2.1>>
- - <<jakarta-jsonb, JSON-B (Jakarta JSON Binding) 1.0>>
- - <<jakarta-jsonp, JSON-P (Jakarta JSON Processing) 1.1>>
- - <<jakarta-annotations, Jakarta Annotations 1.3>>
- - <<mp-config, MicroProfile Config 2.0>>
- - <<mp-fault-tolerance, MicroProfile Fault Tolerance 3.0>>
- - <<mp-health-check, MicroProfile Health 3.1>>
- - <<mp-jwt-auth, MicroProfile JWT Authentication 1.2>>
- - <<mp-metrics, MicroProfile Metrics 3.0>>
- - <<mp-open-api, MicroProfile OpenAPI 2.0>>
- - <<mp-opentracing, MicroProfile OpenTracing 2.0>>
- - <<mp-rest-client, MicroProfile Rest Client 2.0>>
+ - <<jakarta-cdi, CDI (Jakarta Contexts and Dependency Injection) 2.0>> ?
```

Comment:
You need to look up the exact versions of the Jakarta specifications used in Jakarta EE 9.1

Reply:
Do you know where I can look for that ? I guess by observing the Jakarta EE 9.1 dependencies in the POM, right?

Reply:
Updated!!